Zend/zend.c

```diff
@@ -828,7 +828,7 @@ int zend_startup(zend_utility_functions *utility_functions) /* {{{ */
 	{
 		char *tmp = getenv("USE_ZEND_DTRACE");
 
-		if (tmp && zend_atoi(tmp, 0)) {
+		if (tmp && ZEND_STRTOL(tmp, NULL, 0)) {
```

Comment:
Shouldn't the base argument to `ZEND_STRTOL` be `10` (elsewhere as well)?

Reply:
For these changes I chose not to make any unnecessary changes.
If we set "10" here, then ONLY base 10 strings will be respected.
That's probably fine for most of these sites, but I don't have a pressing need to say "You can't use octal/hex".

Reply:
Ah, indeed. I thought zend_atoi() would only accept base-10 numbers (the misnomer is rather confusing).